content/touchpad.md

```diff
@@ -20,9 +20,15 @@ tableOfContents: true
 
 Press <kbd>Fn</kbd>+<kbd>F1</kbd> to turn your laptop touchpad on/off.
 
+>**Note**: This shortcut is <kbd>Fn</kbd>+<kbd>F6</kbd> on the Pangolin (pang12).
+
 To configure your touchpad, press the Super Key <kbd><font-awesome-icon :icon="['fab', 'ubuntu']"></font-awesome-icon></kbd>, <kbd><font-awesome-icon :icon="['fab', 'pop-os']"></font-awesome-icon></kbd> then type <u>Mouse & Touchpad</u> and click on the result. The touchpad settings are under the **Touchpad** tab.
 
-![Mouse Search](/images/touchpad/mouse.png)
+![Mouse Search](/images/touchpad/touchpad-mouse.png)
+
+From here the touchpad can be disabled only while typing like in the screenshot below:
+
+![Mouse & Touchpad Settings](/images/touchpad/touchpad-mouse-settings.png)
 
 <u>GNOME Tweaks</u> has a few other options for the touchpad like disabling it while typing, the acceleration profile and mouse click emulation.
```

Comment:
Part of this sentence is now redundant since disable-while-typing is now mentioned and shown above. (The new screenshot is also redundant with the Tweaks screenshot below this sentence.)

Reply:
Should be fixed in latest commit.

Reply:
Sorry, the sentence did need to be modified, but the screenshot itself wasn't redundant; I didn't realize the new one was of Settings and not GNOME Tweaks since the header bar wasn't included in the new screenshot.

I'm going to change the new Settings one so it includes the window border to better match the Tweaks screenshot, and to highlight the setting being referred to.

Reply:
Hmm, we've actually moved emulation into the main Settings, so that also needs to be removed from the Tweaks callout.

Reply:
Also, I'm not sure why we're explaining how to enable this setting when the setting is already enabled by default.

Reply:
Made some adjustments in https://github.com/system76/docs/pull/1114/commits/ad3a7c73576be9edef29eda2f46d9d642414c9f8 & https://github.com/system76/docs/pull/1114/commits/0abf88634cea88c4acd9b1005cd1ce2379a47350.